.github/workflows/test-rocks-template-repo.yaml

```
@@ -22,17 +22,17 @@ jobs:
 
       - name: Generate rockcraft.yaml from rockcraft init
         run: |
-          mkdir -p rocks2/my-rock/0.1 && cd $_
+          mkdir -p my-rock2/0.1 && cd $_
           rockcraft init
 
       - name: Compare generated and existing rockcraft.yaml
         run: |
           # Remove TODOs from template file
-          sed -i '/^\s*# TODO/d' rocks/my-rock/0.1/rockcraft.yaml
-          sed -i '0,/^\s*$/ { /^\s*$/d }' rocks/my-rock/0.1/rockcraft.yaml
+          sed -i '/^\s*# TODO/d' my-rock/0.1/rockcraft.yaml
+          sed -i '0,/^\s*$/ { /^\s*$/d }' my-rock/0.1/rockcraft.yaml
 
           # Compare the template with the rockcraft init one
-          diff -q rocks/my-rock/0.1/rockcraft.yaml rocks2/my-rock/0.1/rockcraft.yaml
+          diff -q my-rock/0.1/rockcraft.yaml my-rock/0.1/rockcraft.yaml
```

You are comparing the same thing here

```suggestion
          diff -q my-rock/0.1/rockcraft.yaml my-rock2/0.1/rockcraft.yaml
```